firefox_puppeteer/errors.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette.errors import *
```

Comment:
As discussed on IRC we will keep them separate then. The fix will be part of one of the next commits.